lib/issue_badge_application_hooks.rb

```diff
@@ -1,19 +1,18 @@
 class IssueBadgeApplicationHooks < Redmine::Hook::ViewListener
   include ApplicationHelper
 
-  def view_layouts_base_html_head(context = {})
+  def view_layouts_base_html_head(_context = {})
     base_url = Redmine::Utils.relative_url_root
-    badge_url = base_url + "/issue_badge"
+    badge_url = base_url + '/issue_badge'
     o = ''
     if User.current.logged?
-      e = User.current.pref.issue_badge
-      if e == true
-        my_issue_count = Issue.visible.open.where(:assigned_to_id => ([User.current.id] + User.current.group_ids)).size
-        o = stylesheet_link_tag('style', :plugin => 'redmine_issue_badge')
-        o << javascript_include_tag('issue_badge', :plugin => 'redmine_issue_badge')
+      global_enabled = Setting.plugin_redmine_issue_badge['activate_for_all_users'] == 'true'
```

Comment:
__[rubocop]__
* Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)